tests/commands/test_pack.py

```diff
@@ -51,10 +53,66 @@ def func(*, name):
     return func
 
 
-# -- tests for main building process
+# -- tests for the project type decissor
 
 
-def test_simple_succesful_build(tmp_path, caplog, bundle_yaml, config):
+def test_decissor_charm_type(config):
```

Comment:
```suggestion
def test_resolve_charm_type(config):
```

Noun. decisor (plural decisors) (obsolete) a person who resolves or settles matters or controversies

Reply:
So the idea was right. Test the "decisor piece of code", which decides if it's a charm or a bundle.

Taking your suggestions in, anyway.